### PR TITLE
Adding optional property to child button

### PR DESCRIPTION
**Pull request by rodriguezmanu**
https://github.com/ModusCreateOrg/labs/issues/74

### CODE REVIEW

File: src/components/project/DetailButton/index.js [hunk 1]
```diff
@@ -5,17 +5,18 @@ import CloudinaryImage from '../../CloudinaryImage';
...
 
-const ProjectDetailButton = ({ text, className, href, ...rest }) => (
-  <a className={classnames(s.btn, className)} href={href} {...rest}>
+const ProjectDetailButton = ({ text, className, href, child = false, ...rest }) => (
+  <a className={classnames(child ? s.btnChild : s.btn, className)} href={href} {...rest}>
     {text}
-    <CloudinaryImage filename="github.svg" filters={[]} className={s.github} alt="GitHub" />
-    <CloudinaryImage filename="arrow.svg" filters={[]} className={s.arrow} alt={text} />
+    <CloudinaryImage filename="github.svg" filters={[]} className={child ? s.github : s.githubChild} alt="GitHub" />
+    <CloudinaryImage filename="arrow.svg" filters={[]} className={child ? s.arrow: s.arrowChild} alt={text} />
```
Comment (crissoca) [inline src/components/project/DetailButton/index.js:12]: Wouldn't make sense to use `classnames` instead of a ternary? feels cleaner IMO

Comment (rodriguezmanu) [reply src/components/project/DetailButton/index.js]: added @crissoca

File: src/components/project/DetailButton/index.js [hunk 2]
```diff
@@ -5,17 +5,18 @@ import CloudinaryImage from '../../CloudinaryImage';
...
 
-const ProjectDetailButton = ({ text, className, href, ...rest }) => (
-  <a className={classnames(s.btn, className)} href={href} {...rest}>
+const ProjectDetailButton = ({ text, className, href, child = false, ...rest }) => (
+  <a className={classnames(child ? s.btnChild : s.btn, className)} href={href} {...rest}>
     {text}
-    <CloudinaryImage filename="github.svg" filters={[]} className={s.github} alt="GitHub" />
-    <CloudinaryImage filename="arrow.svg" filters={[]} className={s.arrow} alt={text} />
+    <CloudinaryImage filename="github.svg" filters={[]} className={classnames(child ? s.githubChild : s.github)} alt="GitHub" />
+    <CloudinaryImage filename="arrow.svg" filters={[]} className={classnames(child ? s.arrowChild : s.arrow)} alt={text} />
```
Comment (crissoca) [inline src/components/project/DetailButton/index.js:12]: I was thinking in something like:
```suggestion
    <CloudinaryImage
      filename="github.svg"
      filters={[]}
      className={classnames({
        [s.githubChild]: child,
        [s.github]: !child,
      })}
      alt="GitHub"
    />
    <CloudinaryImage
      filename="arrow.svg"
      filters={[]}
      className={classnames({
        [s.arrowChild]: child,
        [s.arrow]: !child,
      })}
      alt={text}
    />
```

File: src/components/project/DetailButton/index.js [hunk 3]
```diff
@@ -5,17 +5,18 @@ import CloudinaryImage from '../../CloudinaryImage';
 
 import s from './styles.module.scss';
 
-const ProjectDetailButton = ({ text, className, href, ...rest }) => (
-  <a className={classnames(s.btn, className)} href={href} {...rest}>
+const ProjectDetailButton = ({ text, className, href, child = false, ...rest }) => (
+  <a className={classnames(child ? s.btnChild : s.btn, className)} href={href} {...rest}>
```
Comment (crissoca) [inline src/components/project/DetailButton/index.js:9]: Maybe this is more readable:
```suggestion
  <a
    className={classnames(className, {
      [s.btnChild]: child,
      [s.btn]: !child,
    })}
    href={href}
    {...rest}
  >
```

**Review by crissoca — COMMENTED** [review]
Added some suggestions.

**Review by crissoca — APPROVED** [review]

**Review by jaygarcia — APPROVED** [review]

**Pull request merged**